internal/core/common/kubelet/client.go

```diff
@@ -74,7 +74,7 @@ func NewClient(kubeletAPI *APIConfig) (*Client, error) {
 		if err != nil {
 			return nil, err
 		}
-		kubeletAPI.URL = fmt.Sprintf("https://%s:10250", hostname)
+		kubeletAPI.URL = fmt.Sprintf("https://%s:10255", hostname)
```

Comment:
`http`.  We should confirm on various platforms.  @asuresh4 can you check on Azure (AKS)?  I will confirm GKE and perhaps @mstumpfx can confirm on EKS since he was working with that recently.